ale/util.py

```diff
@@ -423,3 +426,65 @@ def write_metakernel_from_kernel_list(kernels):
         ])
 
     return body
+
+
+
+def duckpool(naifvar, start=0, length=10, default=None):
+    """
+    Duck typing friendly version of spiceypy kernel pool functions.
+
+    Parameters
+    ----------
+    naifvar : str
+              naif var string to query pool for
+
+    start : int
+            Index of first value
+
+    length : int
+             max number of values returned
+
+    default : obj
+              Default value to return if key is not found in kernel pool
+
+    Returns
+    -------
+    : obj
+      Spice value returned from spiceypy if found, default value otherwise
+
+    """
+    var_exists = spice.expool(naifvar)
+    if var_exists:
+        for f in [spice.gdpool, spice.gcpool, spice.gipool]:
+            try:
+                return f(naifvar, start, length)
+            except:
+                continue
+    return default
+
+
+def duckpond(matchstr="*"):
```

Comment:
Basically to populate naif keywords, instead of manually maintaining a list just grab anything with the instrument's IKID. This way we don't have to discriminate between subtle differences in NAIF keywords and also enforces that the values are those from NAIF.